input/fsh/CodeSystem-Omim.fsh

```diff
@@ -0,0 +1,508 @@
+CodeSystem: OMIM
```

Comment:
It seems like this file is a duplicate?

Reply:
Ugh, changing case in git is messy and I guess I didn't get it right (doesn't help my work is actually done on windows, so case is meaningless...) Will see about fixing that.